build.sbt

```
@@ -5,8 +5,8 @@ lazy val scala212 = "2.12.8"
 ThisBuild / headerLicense  := Some(HeaderLicense.Custom(
   s"""Compiler Interface
      |
-     |Copyright 2011 - 2019, Lightbend, Inc.
-     |Copyright 2008 - 2010, Mark Harrah
+     |Copyright 2011-2020, Lightbend, Inc.
+     |Copyright 2008-2010, Mark Harrah
```

```suggestion
     |Copyright Mark Harrah and Lightbend, Inc.
```

Maybe we can drop the year range, like we did with scala/scala.

---

good suggestion, will do